umd-federation/e2e/checkUmdFederationApps.cy.ts

```diff
@@ -0,0 +1,54 @@
+import {BaseMethods} from "../../cypress/common/base";
+import {baseSelectors} from "../../cypress/common/selectors";
+import {Constants} from "../../cypress/fixtures/constants";
+const basePage: BaseMethods = new BaseMethods()
+
+const appsData = [
+    {
+        header1: Constants.elementsText.umdFederationHeader1,
+        header2: Constants.elementsText.umdFederationHeader2,
+        header3: Constants.elementsText.umdFederationHeader3,
+        host: 9001
+    },
+
+    {
+        header1: Constants.elementsText.umdFederationHeader1,
+        header2: Constants.elementsText.umdFederationHeader2,
+        header3: Constants.elementsText.umdFederationHeader3,
+        host: 9002
+
+    }
+]
+
+appsData.forEach((
+    property: {
+        header1: string,
+        header2: string,
+        header3: string,
+        host: number
+    }
+) => {
+    describe(`Check app1 and app2 starts and running`, () => {
+        beforeEach(()=> {
+            basePage.openLocalhost(property.host)
+        })
+        it (`Check App1 elements`, () => {
+            basePage.skipTestByCondition(property.host === 9002)
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h1,
+                text: String(property.header1)
```

Comment:
That's why I am asking you to remove the `header1, header2, header3` from app2 object and make them optional cause if you deside to left them you don't need String(...) method here and you don't get any errors here, but if you remove them from app2 object and make them optional without String(...) you will have an error there, that your attribute isn't a string)